testkit-backend/backend.go

```diff
@@ -112,7 +112,8 @@ func (b *backend) writeError(err error) {
 	isDriverError := neo4j.IsNeo4jError(err) ||
 		neo4j.IsUsageError(err) ||
 		neo4j.IsConnectivityError(err) ||
-		neo4j.IsTransactionExecutionLimit(err)
+		neo4j.IsTransactionExecutionLimit(err) ||
+		err.Error() == "Invalid transaction handle"
```

Comment:
It seems to me this error should be created as a usage error, not an ad-hoc one as it is currently the case.
Given the current implementation, this seems like the way to go.